deployments/_deployments_log_file.json

```diff
@@ -31761,6 +31761,146 @@
           }
         ]
       }
+    },
+    "berachain": {
+      "production": {
+        "1.0.0": [
+          {
+            "ADDRESS": "0x2E4E7beD12E1EeE5E98582EE41aDf36Ce40d2dE4",
+            "OPTIMIZER_RUNS": "1000000",
+            "TIMESTAMP": "2025-05-07 17:14:21",
+            "CONSTRUCTOR_ARGS": "0x000000000000000000000000023fa838682c115c2cfba96ef3791cb5bd931fc7",
+            "SALT": "",
+            "VERIFIED": "true"
+          }
+        ]
+      }
+    },
+    "blast": {
+      "production": {
+        "1.0.0": [
+          {
+            "ADDRESS": "0xf34B594fDBC0dD8477756Cb7C6e77CF9A71691d6",
+            "OPTIMIZER_RUNS": "1000000",
+            "TIMESTAMP": "2025-05-07 16:28:50",
+            "CONSTRUCTOR_ARGS": "0x000000000000000000000000023fa838682c115c2cfba96ef3791cb5bd931fc7",
+            "SALT": "",
+            "VERIFIED": "true"
```

Comment:
contract is not verified. Should we leave it?

Reply:
https://blastscan.io/address/0xf34B594fDBC0dD8477756Cb7C6e77CF9A71691d6#code